ObjectiveSee/DoNotDisturb.munki.recipe

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads, unzips, verifies, packages and imports the installer of the latest version of DoNotDisturb by Objective-See into munki.</string> 
+    <key>Identifier</key>
+    <string>com.github.its-unibas.munki.DoNotDisturb</string>
+    <key>ParentRecipe</key>
+    <string>com.github.its-unibas.pkg.DoNotDisturb</string>
+    <key>MinimumVersion</key>
+    <string>1.0.0</string>
+    <key>Input</key>
+    <dict>
+        <key>MUNKI_NAME</key>
+        <string>%NAME%</string>
+        <key>MUNKI_REPO_SUBDIR</key>
+        <string>apps/%NAME%</string>
+        <key>MUNKI_CATEGORY</key>
+        <string>Utilities</string>
+        <key>MUNKI_DEVELOPER</key>
+        <string>Objective-See</string>
+        <key>DISPLAY_NAME</key>
+        <string>%NAME%</string>
+        <key>pkginfo</key>
+        <dict>
+            <key>catalogs</key>
+            <array>
+                <string>testing</string>
+            </array>
+            <key>description</key>
+            <string>Do Not Disturb (DND) is a free, open-source utility that aims to detect and alert you of "evil maid" attacks</string>
+            <key>display_name</key>
+            <string>%DISPLAY_NAME%</string>
+            <key>name</key>
+            <string>%MUNKI_NAME%</string>
+            <key>unattended_install</key>
+            <true/>
+            <key>category</key>
+            <string>%MUNKI_CATEGORY%</string>
+            <key>developer</key>
+            <string>%MUNKI_DEVELOPER%</string>
+            <key>postinstall_script</key>
+            <string>#!/bin/sh
+sudo "/Applications/Do Not Disturb Installer.app/Contents/MacOS/%NAME% Installer"
+rm -rf "/Applications/%NAME% Installer.app/"
+exit 0
+            </string>
+            <key>preuninstall_script</key>
```

Comment:
Please quote the source for this script (https://objective-see.com/products/dnd.html)
Also, user `#` for comments instead of `//` to be consistent with our codebase.

Reply:
It would be great if you could add a comment about why the steps provided in https://objective-see.com/products/dnd.html are still needed when using a pkg for install/uninstall.

Reply:
I have now implemented the requested changes.